lib/countly.js

```diff
@@ -1832,8 +1842,25 @@
         * @param {number} feedback.rating - user's rating from 1 to 5
         * @param {string=} feedback.email - user's email
         * @param {string=} feedback.comment - user's comment
+        * 
+        * @deprecated use 'recordRatingWidgetWithID' in place of this call
         **/
         this.report_feedback = function(feedback) {
+            log("report_feedback, Deprecated function call! Use 'recordRatingWidgetWithID' in place of this call. Call will be redirected now!");
+            this.recordRatingWidgetWithID(feedback);
+        };
+        /**
+        * Provide information about user
+        * @param {Object} feedback - object with feedback properties
```

Comment:
rename the function variable to "ratingWidget" and respectively also the name used in code comments from "feedback" to "ratingWidget".